catalog/spatial/wfs/2.0.0/spatial-wfs-v2_0_0-source/src/main/java/org/codice/ddf/spatial/ogc/wfs/v2_0_0/catalog/source/WfsResponseExceptionMapper.java

```diff
@@ -86,9 +92,9 @@ public WfsException fromResponse(Response response) {
   private WfsException convertToWfsException(ExceptionReport report) {
 
     WfsException wfsException = null;
-    List<ExceptionType> list = new ArrayList<ExceptionType>(report.getException());
+    List<ExceptionType> list = new ArrayList<>(report.getException());
```

Comment:
❓ Why not clone the list within the if t avoid creating an empty list for nothing?